app/help.py

```diff
@@ -96,5 +96,6 @@
   See readme.md for an introduction
   Both files may be found in "%s"
   Please send feedback and bug reports to dschuyler@
+  (Release build)
```

Comment:
Oh sorry, I think I made it branch from the release branch because I didn't know which of the branches I was going to merge into. I'll start making everything based on the master from now on! 👍